src/etc/_cargo

```diff
@@ -444,4 +444,8 @@ _cargo_example_names() {
     fi
 }
 
+_cargo_yesno() {
+    _values 'yes|no' yes no
```

Comment:
I don't know if I did this right and don't have any idea on how to actually test this. I tried to read the zsh docs on the compdef file format but... wow it's dense. Any guidance would be appreciated.